### DISCUSSION TITLE
<title>Github Octernship

### DISCUSSION THREAD

**Discussion by 20santi** [community/community#106318] [GitHub Education]
### Select Topic Area

General

### Body

Can anyone tell me how can i apply in octernship. I can not find any section for apply

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @20santi , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!

**Comment by mishmanners** [comment]
GitHub Octnerships are currently unavailable. You can keep an eye out as to when they become available again on the [GitHub Global Campus website](https://education.github.com/). You can also check out the [repo](https://github.com/education/Octernships) for information on the program, and you'll notice the program is currently not accepting applications.
